insalan/mailer.py

```diff
@@ -0,0 +1,255 @@
+import logging
+import sys
+from django.contrib.auth.models import Permission
+from django.core.mail import EmailMessage, get_connection, send_mail
+from django.contrib.auth.tokens import (
+    PasswordResetTokenGenerator,
+    default_token_generator,
+)
+from django.utils.translation import gettext_lazy as _
+
+import insalan.settings
+from insalan.user.models import User
+from insalan.tickets.models import TicketManager
+from apscheduler.schedulers.background import BackgroundScheduler
+
+class EmailConfirmationTokenGenerator(PasswordResetTokenGenerator):
+    """
+    Generate an email confirmation token.
+    It's just a PasswordResetTokenGenerator with a different salt.
+
+    (NB: the django app secret is also used as a salt)
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.key_salt = "IWontLaunch8TwitchStreamsWhenConnectionIsAlreadyBad"
+
+
+class UserMailer:
+    """
+    Send emails.
+    """
+    def __init__(self, MAIL_HOST:str, MAIL_PORT:str, MAIL_FROM: str, MAIL_PASS: str, MAIL_SSL:bool, TEST: bool = False):
```

Comment:
not a fan of capitals used for arguments here, or for variables (in python it's more commonly used for class attributes aka static fields common to all elements of a class)